platform/osx/export/export_plugin.cpp

```diff
@@ -777,6 +777,32 @@ Error EditorExportPlatformOSX::export_project(const Ref<EditorExportPreset> &p_p
 		err = tmp_app_dir->make_dir_recursive(tmp_app_path_name + "/Contents/Resources");
 	}
 
+	Vector<String> translations = ProjectSettings::get_singleton()->get("internationalization/locale/translations");
+	if (translations.size() > 0) {
+		{
+			String fname = tmp_app_path_name + "/Contents/Resources/en.lproj";
+			tmp_app_dir->make_dir_recursive(fname);
+			FileAccess *f = FileAccess::open(fname + "/InfoPlist.strings", FileAccess::WRITE);
```

Comment:
Would `FileAccessRef` work here too? It's nicer not to have to do close + memdelete manually.

Reply:
Yes, it will work, changed to `FileAccessRef`.